api/api.go

```diff
@@ -398,8 +398,16 @@ type CloudBackupGenericRequest struct {
 	// MetadataFilter indicates backups whose metadata has these kv pairs
 	MetadataFilter map[string]string
 	// CloudBackupID must be specified if one needs to enumerate known single
-	// backup( format is clusteruuidORBucketName/srcVolId-SnapId(-incr)
+	// backup( format is clusteruuidORBucketName/srcVolId-SnapId(-incr). If t\
+	// this is specified, everything else n the command is ignored
 	CloudBackupID string
+	// MissingSrcVol set to true enumerates cloudbackups for which srcVol is not
```

Comment:
nit: prefer to use full names when possible.
s/srcVol/SrcVolumeID/

Redo the comment ? The '(' does not end ?

Reply:
Since the field does not represent ID changed it to MissingSrcVolumes